### DISCUSSION TITLE
comma into arrays

### DISCUSSION THREAD

**Discussion by SamueleD78** [vfsfitvnm/frida-il2cpp-bridge#236] [Q&A] [answered]
So, i understood how to use classes array using `arrayClass`.

Now, into the classes dump i have a class with this prototype:

```
    Class.Someclass[,][] m_somename;
```

That comma is really weird..
How this kind of arrays are instantiated/accessed?

**Comment by vfsfitvnm** [comment]
https://stackoverflow.com/questions/13807493/what-does-it-mean-when-there-is-a-comma-inside-an-array-declaration-e-g-float

**Reply by SamueleD78** [reply]
ok but how i can instantiate/access one with frida-il2cpp-bridge?

**Comment by SamueleD78** [comment, marked as answer]
sorry to insist, but is there a way with frida-il2cpp-bridge to instatiate an array with the [,] multidimensional specification?

Is it just a .arrayClass.arrayClass?

thank you in advance

**Reply by vfsfitvnm** [reply]
yes it is (I guess)

**Reply by SamueleD78** [reply]
that's reassuring XD
thank you anyway ^^
